chain-test/src/unit/FabricIterable.ts

```diff
@@ -14,6 +14,26 @@
  */
 import { Iterators } from "fabric-shim";
 
+/**
+ * Utilities for creating and manipulating async iterators compatible with Hyperledger Fabric.
```

Comment:
this file is actually a copy of `chaincode/src/types/FabricIterable.ts`, so all comments can be applied there.

this is a workaround since:
1. `FabricIterable` is required for our cache - which is included in the `chaincode` package
2. it is also required to make e2e mocked tests work
3. we don't want `test` package to depend on `chaincode`
4. we don't want `chaincode` to depend on `test` (non-dev dep)

so we have a circle here, and tbh I still don't know how to approach it